cmd/local.go

```diff
@@ -7,42 +7,33 @@ import (
 	"github.com/spf13/viper"
 	"log/slog"
 	"os"
-	"os/user"
-	"runtime"
 )
 
 type localCmdOptions struct {
-	useDKMS bool
-	srcDir  string
-	envMap  map[string]string
+	useDKMS         bool
+	downloadHeaders bool
```

Comment:
Local cmd now supports automatically downloading kernel headers for the build.
Note that it is not a fatal error if kernel headers are not found; it will just continue trying with locally installed ones, if any.